src/LineBot/LineBot.cs

```diff
@@ -198,9 +198,9 @@ public async Task<IEnumerable<ILineEvent>> GetEvents(HttpRequest request)
             var eventCollection = JsonConvert.DeserializeObject<LineEventCollection>(jsonContent);
 
             if (eventCollection == null || eventCollection.Events == null)
-                return Enumerable.Empty<ILineEvent>();
+                return LineEvents.Empty();
 
-            return eventCollection.Events;
+            return new LineEvents();
```

Comment:
This is probably why your tests are failing. You are returning an empty object here.

Reply:
Umm... we should be returning an empty object right? because it was returning an empty object before.

Reply:
You should return the `Events` here instead. When you just do `new LineEvents()` you return an object without any events.

Reply:
> Sorry my bad, I get it what you mean.